src/cache/algorithms/expr_variable_reordering.cpp

```diff
@@ -0,0 +1,253 @@
+// Rafael Sá Menezes - 04/2020
+
+#include "expr_variable_reordering.h"
+
+#include <cache/algorithms/algorithms_utils.h>
+
+namespace
+{
+constexpr std::array<expr2t::expr_ids, 4> binary_operations = {
+  expr2t::expr_ids::add_id,
+  expr2t::expr_ids::sub_id,
+  expr2t::expr_ids::mul_id,
+  expr2t::expr_ids::div_id};
+
+constexpr std::array<expr2t::expr_ids, 6> relations = {
```

Comment:
Why "6"? Add description to the magic numbers.

Reply:
std::array needs the number of elements of the array